lib/index.js

```diff
@@ -32,11 +32,12 @@ const tmpTsconfig = {
 fs.writeFileSync(tmpTsconfigPath, JSON.stringify(tmpTsconfig, null, 2))
 
 // Type-check our files
-spawnSync(
+const { status } = spawnSync(
   resolveFromModule('typescript', 'bin/tsc'),
   ['-p', tmpTsconfigPath, '--noEmit'],
   { stdio: 'inherit' },
 )
 
 // Delete temp config file
 fs.unlinkSync(tmpTsconfigPath)
+process.exit(status);
```

Comment:
Thanks for the PR! Please add an empty line above the `process.exit(status)` and rename the commit to `feat: exit with tsc's status`.

Reply:
BTW I moved to the alpha branch because we want `semantic-release` to release the version with `-alpha.x` suffix.

Reply:
I've done the changes myself. Thanks again for the PR. :)